sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexClient.java

```diff
@@ -20,7 +20,10 @@
 import com.azure.search.documents.indexes.models.SynonymMap;
 
 /**
- * Synchronous Client to manage and query indexes, as well as manage Synonym Map, on a Cognitive Search service
+ * This class provides a client that contains the operations for creating, getting, listing, updating, or deleting
```

Comment:
Do we need to mention it is a synchronous client?

Reply:
Possibly, not completely certain. Some SDKs do call out synchronous vs asynchronous while others do not. I think we'll want to decide this at a higher level. For now let's move forward not calling it out.